identity-apps-core/apps/recovery-portal/src/main/webapp/accept-invitation.jsp

```diff
@@ -0,0 +1,213 @@
+<%@ page import="java.net.HttpURLConnection, java.net.URL, java.io.BufferedReader, java.io.InputStreamReader" %>
+<%@ page import="java.io.OutputStream" %>
+<%@ page import="org.wso2.carbon.identity.mgt.endpoint.util.IdentityManagementEndpointConstants" %>
+<%@ page import="java.io.File" %>
+<%@ page import="org.wso2.carbon.identity.mgt.endpoint.util.IdentityManagementEndpointUtil" %>
+<%@ taglib prefix="layout" uri="org.wso2.identity.apps.taglibs.layout.controller" %>
+
+<%-- Localization --%>
+<jsp:directive.include file="includes/localize.jsp"/>
+
+<%-- Include tenant context --%>
+<jsp:directive.include file="tenant-resolve.jsp"/>
+
+<%-- Branding Preferences --%>
+<jsp:directive.include file="includes/branding-preferences.jsp"/>
+
+<%
+    // URL of the REST API you want to call
+    String apiUrl = "https://api.asg.io/api/server/v1/guests/invitation/accept";
+    String confCode = request.getParameter("code");
+    String acceptApiResponse = "";
+    
+    try {
+        // Create a URL object with the API URL
+        URL url = new URL(apiUrl);
+        
+        // Open a connection to the URL
+        HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+        
+        // Set the HTTP request method to POST
+        connection.setRequestMethod("POST");
+
+        // Set the request content type (e.g., JSON)
+        connection.setRequestProperty("Content-Type", "application/json");
+
+        // Enable input/output streams for sending and receiving data
+        connection.setDoOutput(true);
+        connection.setDoInput(true);
+
+        // Define the request body content (e.g., JSON data)
+        String requestBody = "{\"confirmationCode\":\"" + confCode + "\"}";
+        // out.println("API Request body: " + requestBody);
+
+        // Write the request body data to the connection's output stream
+        try (OutputStream os = connection.getOutputStream()) {
+            byte[] input = requestBody.getBytes("UTF-8");
+            os.write(input, 0, input.length);
+        }
+        
+        // Get the response code from the server
+        int responseCode = connection.getResponseCode();
+        
+        // Check if the response code is successful (e.g., 200 OK)
+        if (responseCode == 204) {
+            // Read the API response
+            BufferedReader reader = new BufferedReader(new InputStreamReader(connection.getInputStream()));
+            StringBuilder response2 = new StringBuilder();
+            String line;
+            while ((line = reader.readLine()) != null) {
+                response2.append(line);
+            }
+            reader.close();
+            
+            // Now you have the API response in the 'response' StringBuilder
+            String apiResponse = response2.toString();
+            acceptApiResponse="SUCCESS";
+            // You can process and display the API response as needed
+            // out.println("API Response: " + apiResponse);
```

Comment:
Can we remove redundant print statements?

Reply:
Done